akka-actor-typed/src/main/scala/akka/actor/typed/ActorRef.scala

```diff
@@ -66,22 +64,10 @@ object ActorRef {
    * Create an ActorRef from a Future, buffering up to the given number of
    * messages in while the Future is not fulfilled.
    */
-  def apply[T](f: Future[ActorRef[T]], bufferSize: Int = 1000): ActorRef[T] =
+  private[akka] def apply[T](f: Future[ActorRef[T]], bufferSize: Int = 1000): ActorRef[T] =
```

Comment:
add the full internal markers
```
/**
   * INTERNAL API
   */
  @InternalApi private[akka] 
```

Reply:
Updated

Reply:
Does not seem to have been updated? Missing InternalApi annotation and comment